doc/Repeater.md

```diff
@@ -1,8 +1,63 @@
+# Quantum Repeater
 
-*Note: this file is an early design document, but might be updated as the version goes up.*
+## What is this?
 
-# Repeater #
+Quantum Repeater is one crucial network device in entanglement-based quantum networking, which allows us to extend link-level entanglement to end-to-end entanglement. 
```

Comment:
Based on our network architecture paper, a repeater has exactly two interfaces. I think the QuISP code currently makes no such distinction?

Reply:
Repeater always has two QNICs in QuISP. Is that what you mean here?

Reply:
Yes.

Reply:
Although one question is, if internally the Repeater has two QNICs, how do we create entanglement between the QNICs? That's not really needed in a lot of Repeater implementations, though it is Router implementations that use an optical switch and multiple independent line cards.
In a Router, each QNIC has two optical connections -- one to the wide-area fiber, one to the optical backplane. So a Repeater is effectively a standalone linecard.